assets/home/tasks-section/index.test.js

```diff
@@ -43,4 +48,76 @@ describe( '<TasksSection />', () => {
 			)
 		).toBeTruthy();
 	} );
+
+	// it( 'Should post to the server that all tasks are completed', async () => {
```

Comment:
Unfortunately, After adding the `setTimeout` to improve the transition to the ready state, I needed to comment both tests because jest timers aren't working correctly with `waitFor`.